packages/core/src/types.ts

```diff
@@ -5,12 +5,50 @@ import {
 } from '@docknetwork/wallet-sdk-data-store/src/types';
 import {EventEmitter} from 'events';
 
+/**
+ * Legacy V1 wallet interface for backward compatibility
```

Comment:
are we okay with devs referencing this, or should we mark it as obsolete or something so they don't start adding dependencies on it?